From e87c2ad1e16cdbd62ba71b2ace3270503decaa56 Mon Sep 17 00:00:00 2001
From: "Mingde (Matthew) Zeng" <matthewzmd@gmail.com>
Date: Wed, 29 Jul 2020 08:47:09 -0400
Subject: [PATCH] Remove OOM tests from runtest/mm

Disable OOM tests, as they might cause oeqa ssh connection lost

Upstream-Status: Inappropriate [oe-core specific]
Signed-off-by: Mingde (Matthew) Zeng <matthew.zeng@windriver.com>
[ pvorel: rebased for 20210927 ]
Signed-off-by: Petr Vorel <petr.vorel@gmail.com>
---
 runtest/mm | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/runtest/mm b/runtest/mm
index 6537666a9..e5a091a5a 100644
--- a/runtest/mm
+++ b/runtest/mm
@@ -73,12 +73,6 @@ ksm06_2 ksm06 -n 8000
 
 cpuset01 cpuset01
 
-oom01 oom01
-oom02 oom02
-oom03 oom03
-oom04 oom04
-oom05 oom05
-
 swapping01 swapping01 -i 5
 
 thp01 thp01 -I 120
-- 
2.33.0

